### PR TITLE
codec_adpater: change the buffer size of waves_codec

### PR DESCRIPTION
**Pull request by CyKuei**
Change the buffer size of waves_codec from 2ms to SCHEDULE_PERIOD

(Please refer to https://github.com/thesofproject/sof/pull/5851)

Signed-off-by: CY Kuei <cyk@waves.com>

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test&nbsp;this&nbsp;please** to run this test once

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

File: src/audio/codec_adapter/codec/waves.c
Comment (dbaluta) [inline src/audio/codec_adapter/codec/waves.c:331]: Can you please mention why we need this change in the commit message?

Comment (lgirdwood) [reply src/audio/codec_adapter/codec/waves.c:331]: We found out that this component ignore the topology setting, this PR backports the fix to a stable Intel product release.

**Review by lgirdwood — APPROVED** [review]
@mwasko @sathya-nujella fyi - non topology fix.

**Review by dbaluta — APPROVED** [review]

**Comment by lgirdwood** [comment]
@mwasko is on vacation. @marcinszkudlinski who should take care of production branch this week ?

**Comment by lgirdwood** [comment]
@aborisovich @abonislawski who should approve this for production branch when @mwasko is out ?

**Review by cujomalainey — APPROVED** [review]

**Comment by lgirdwood** [comment]
@mwasko ping

**Pull request merged**
